SpywareFilter/sections/specific.txt

```diff
@@ -101,6 +101,7 @@ news.mynavi.jp##.articleList-attention
 ||hondacarsrockhill.com/pixall/pix-ddc-fp.min.js
 ||servebom.com/event.js
 ||flipkart.net/*/*/tracker/
+/magazine-newspaper-pro/analytics.js$domain=infolaw.co.uk
```

Comment:
I have a question from https://github.com/AdguardTeam/AdguardFilters/pull/83113#discussion_r631675647
If you have a different status for this request, please provide its contents, otherwise this blocking is not necessary.

Reply:
Yeah, you are right. But should the featured services be blocked? It's currently unblocked and it leads to some webpages.
![image](https://user-images.githubusercontent.com/76908410/118110301-d4f36a80-b414-11eb-9d92-2ae9de6d4644.png)

Reply:
This is rather doubtful, but maybe for Annoyances. These are their partners, but on these sites I can't find any information about their partnership.

This for Annoyances - annoyances.txt
```
infolaw.co.uk#?#.inside-sidebar h3:contains(Featured services)
infolaw.co.uk#?#.inside-sidebar h3:contains(Featured services) ~ p
```
Also we can block their socials `! ##.twitter-timeline + widgets.js`.
```
||platform.twitter.com/widgets.js$domain=infolaw.co.uk
infolaw.co.uk##.twitter-timeline
```